validator/src/main.rs

```diff
@@ -1546,6 +1546,12 @@ pub fn main() {
                 .help("Enables faster starting of validators by skipping shrink. \
                       This option is for use during testing."),
         )
+        .arg(
+            Arg::with_name("no_serve_snapshots")
+                .long("no-serve-snapshots")
+                .help("Inhibit serving on snapshots. \
```

Comment:
maybe "Disable publishing snapshots"? I think I'd get rid of the 'for use...'. A comment is good that says that.